lib/yubihsm_winhttp.c

```diff
@@ -102,9 +102,9 @@ static bool parseUrl(char *url, struct urlComponents *components) {
   return true;
 }
 
-static void CALLBACK http_callback(HINTERNET internet __attribute__((unused)),
-                                   DWORD_PTR context, DWORD status,
-                                   LPVOID statusInfo, DWORD statusInfoLen) {
+static void CALLBACK http_callback(HINTERNET internet, DWORD_PTR context,
```

Comment:
doesn't unused work on msvc? this will generate a new warning then I guess? We have an UNUSED macro in yubihsm.h that can be used in that case.

Reply:
No, that caused an error on MSVC. Replaced it now with UNUSED macro